src/components/utils/test/thread_test.cc

```diff
@@ -74,14 +78,18 @@ class ThreadTestDelegate : public threads::ThreadDelegate {
   void WaitForThreadStart() {
     AutoLock test_lock(test_mutex_);
     while (!thread_started_) {
-      cond_var_.Wait(test_lock);
+      cond_var_.WaitFor(test_lock, kMilliSecondsToWait);
     }
   }
 
   bool is_thread_started() const {
     return thread_started_;
   }
 
+  // Desired value should equal 1
+  int is_thread_restarted() const {
```

Comment:
@wolfylambova22 It is better to change returned value to bool and retun just:
return starts_counter_ == 1;
because I do not see any other type of usage for it.